### ISSUE TITLE
Add friendlyName inspection to validation package

### ISSUE THREAD

**Issue by ngrices** [ngrices/alexa-smarthome-sdk#1]
Idea from a partner, add friendlyName validation to validation package in each language to avoid discovery failures due to naming.
We should define the charset for each language that would invalidate the friendlyName field.

**Comment by dairong** [comment]
Agreed, but we also need to get the valid set of chars from the SH team. I think we can focus on US first, get it working well with some partners, then incrementally add code for other languages.
